src/aks-preview/azcli_aks_live_test/configs/cli_matrix_default.json

```diff
@@ -5,10 +5,16 @@
         ]
     },
     "exclude": {
+        "compatibility issue": [
+            "test_aks_nodepool_add_with_ossku"
+        ],
         "need additional feature": [
             "test_aks_create_enable_encryption",
             "test_aks_create_edge_zone",
-            "test_aks_create_with_auto_upgrade_channel"
+            "test_aks_create_with_auto_upgrade_channel",
```

Comment:
did you file work items to track this?

Reply:
Good catch, now we have work items to track these cases.